packages/pwa-kit-react-sdk/src/utils/opentelemetry.js

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {trace, context, SpanStatusCode} from '@opentelemetry/api'
+import {hrTimeToMilliseconds, hrTimeToTimeStamp} from '@opentelemetry/core'
+import logger from './logger-instance'
+
+const SERVICE_NAME = 'pwa-kit-react-sdk'
+
+function logSpanData(span, event = 'start', res = null) {
```

Comment:
Can we use the arrow function pattern used throughout the codebase instead of regular function declarations for consistency?
```suggestion
const logSpanData = (span, event = 'start', res = null) => {
```